### ISSUE TITLE
Qt Icons have a darker gray background and have fixed size

### ISSUE THREAD

**Issue by addy419** [awesomeWM/awesome#3360]
Output of `awesome --version`:

```
awesome 4.3 (Too long)
 • Compiled against Lua 5.2.4 (running with Lua 5.2)
 • D-Bus support: ✔
 • execinfo support: ✔
 • xcb-randr version: 1.6
 • LGI version: [string "return require('lgi.version')"]:1: module 'lgi.version' not found:
	no field package.preload['lgi.version']
	no file '/usr/share/lua/5.2/lgi/version.lua'
	no file '/usr/share/lua/5.2/lgi/version/init.lua'
	no file '/usr/lib/lua/5.2/lgi/version.lua'
	no file '/usr/lib/lua/5.2/lgi/version/init.lua'
	no file './lgi/version.lua'
	no file '/usr/lib/lua/5.2/lgi/version.so'
	no file '/usr/lib/lua/5.2/loadall.so'
	no file './lgi/version.so'
	no file '/usr/lib/lua/5.2/lgi.so'
	no file '/usr/lib/lua/5.2/loadall.so'
	no file './lgi.so'
```
Since I'm using NixOS, the lgi issue is due to the fact that --version does not consult to --search https://discourse.nixos.org/t/awesomewm-luamodules-apparently-not-taking-effect/8507/2

**How to reproduce the issue:**

Create a systray with any background color except `#1F212B`, Qt icons will have `#1F212B` as background.

**Actual result:**
https://imgur.com/5g9M27q

I also tried changing the `systray:set_base_size()` to 30 and the gtk icons responded to this but not Qt
https://imgur.com/j2zOmNy

In some previous issues, it is stated that this is a gtk bug, but background should opaque for both gtk and qt icons then, this affects all the qt apps I use, namely nm-tray, wpa_supplicant_gui (tray), kdeconnect-indicator

**Comment by actionless** [comment]
it's a duplicate of thousand others issues about broken tray in qt

**Comment by actionless** [comment]
tl;dr - if you really need it, install some 3rd-party tray app, and force its placement in the area of the screen you want

**Comment by addy419** [comment]
https://github.com/awesomeWM/awesome/issues/363#issuecomment-142048007 I searched the issues that state this is a XEmbed issue. But KdeConnect indicator follows the AppIndicator specification, why is it facing the same issue?
